Python/ceval_macros.h

```diff
@@ -246,6 +246,8 @@ GETITEM(PyObject *v, Py_ssize_t i) {
 #define STACK_SHRINK(n)        BASIC_STACKADJ(-(n))
 #endif
 
+#define  WITHIN_STACK_BOUNDS() \
```

Comment:
```suggestion
#define WITHIN_STACK_BOUNDS() \
```
Two spaces seem redundant